### PR TITLE
update: adding more type of cards

### PR DESCRIPTION
**Pull request by ultronjs**
a.)Adding more types of card in card component. Addition contains
1.)Card with Badge
2.)Card with Text Only
3.)Card with Text Overlay
4.)Card with Dismiss.
b.)add card.js also for functioning of dismiss card
c.)Also mode the hide class from badge css to global level i.e in display css

### CODE REVIEW

**Review by ozairkhan2810 — APPROVED** [review]
Code looks good.

File: css/components/card.css [hunk 1]
```diff
@@ -38,6 +38,27 @@ Actions contains:-Button and Icons*/
...
+/* For Only Text cards*/
+.card_only_text{
+    height: fit-content;
+}
+.card_with_dismiss{
+    position: relative;
+}
+
+.card_dismiss{
+    padding: 0.4rem 0.2rem;
```
Comment (ShreyashMahajan) [inline css/components/card.css:50]: you can use px for less than 1rem

**Review by ShreyashMahajan — APPROVED** [review]
naming used for code is very understandable.

File: css/components/card.css (hunk 1 above)
Comment (ultronjs) [reply css/components/card.css:50]: Thanks, good suggestion. I have used this at many places. Will do these changes in refactoring pr

**Comment by ultronjs** [comment]
Merging

**Pull request merged**